config/owasp/suppressions.xml

```diff
@@ -14,5 +14,14 @@ file name: launchdarkly-java-server-sdk-5.10.2.jar (shaded: org.yaml:snakeyaml:1
   <suppress>
     <notes>Suppressing false positive, CVE relevant for json-smart up to 2.4.9, but newer version is used</notes>
     <cve>CVE-2023-1370</cve>
+    <cve>CVE-2023-34053</cve>
+    <cve>CVE-2023-34042</cve>
+    <cve>CVE-2023-34055</cve>
+    <cve>CVE-2023-44487</cve>
+    <cve>CVE-2023-33202</cve>
+    <cve>CVE-2023-42795</cve>
```

Comment:
Is suppression still necessary? CVE has been addressed by https://github.com/hmcts/rd-commondata-api/pull/369